CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+
+## The contribution process
+
+_Pull request early_
+
+We encourage you to create pull requests early. It helps us track the contributions under development, whether they are ready to be merged or not. Tag your pull request as `[WIP]` until it is ready for formal review.
+
+
+### Submitting pull requests
+All code changes to the master branch must be done via pull requests.
+1. Create a new ticket or take a known ticket from [the issue list][monai issue list].
+1. Check if there's already a branch dedicated to the task.
+1. If the task has not been taken, [create a new branch](https://help.github.com/en/desktop/contributing-to-projects/creating-a-branch-for-your-work) in the codebase or a fork of the codebase named `[ticket_id]-[task_name]`. For example, branch name `19-ci-pipeline-setup` corresponds to [issue #19](https://github.com/Project-MONAI/MONAI/issues/19).
+Ideally, the new branch should be based on the latest `master` branch.
+1. Make changes to the branch ([use detailed commit messages if possible](https://chris.beams.io/posts/git-commit/)). If the changes introduce new features, make sure that you write [unit tests](#unit-testing).
+1. [Create a new pull request](https://help.github.com/en/desktop/contributing-to-projects/creating-a-pull-request) from the task branch to the master branch, with detailed descriptions of the purpose of this pull request.
+1. Check [the CI/CD status of the pull request][gitlab ci], make sure all CI/CD tests passed.
+1. Wait for reviews; if there are reviews, make point-to-point responses, make further code changes if needed.
+1. If there're conflicts between the pull request branch and the master branch, pull the changes from the master and resolve the conflicts locally .
+1. Reviewer and contributor may have discussions back and forth until all comments addressed.
+1. Wait for the pull request to be merged.
+
+### Coding style
+Coding style is checked by flake8, using [a flake8 configuration](https://github.com/Project-MONAI/MONAI/blob/master/.flake8) similar to [PyTorch's](https://github.com/pytorch/pytorch/blob/master/.flake8).
+
+License information: all source code files should start with this paragraph:
+```
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+```
+
+## Unit testing
```

Comment:
am i correct on the unit testing? please feel free to push to this branch directly if im wrong @ericspod